providers/MomentCreateProvider.tsx

```diff
@@ -0,0 +1,32 @@
+"use client";
+
+import useZoraCreate from "@/hooks/useZoraCreate";
+import React, { createContext, useContext, useMemo } from "react";
+
+const MomentCreateContext = createContext<
+  ReturnType<typeof useZoraCreate> | undefined
+>(undefined);
+
+const MomentCreateProvider = ({ children }: { children: React.ReactNode }) => {
+  const zoraCreate = useZoraCreate();
```

Comment:
TODO: refactor `useZoraCreate` => `useCreateMoment`
- make a new ticket for this